airflow/hooks/hive_hooks.py

```diff
@@ -343,7 +343,8 @@ def load_file(
             create=True,
             overwrite=True,
             partition=None,
-            recreate=False):
+            recreate=False,
+            tblproperties={}):
```

Comment:
I'm gonna recommend not doing tblproperties={}, better off doing None. The reason is that the same dictionary is shared across all invocations which could cause problems later.